scripts/ci/gke.sh

```diff
@@ -1,16 +1,13 @@
 #!/usr/bin/env bash
 
-set -euo pipefail
-
 # A collection of GKE related reusable bash functions for CI
 
-set +u
 SCRIPTS_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")"/../.. && pwd)"
-set -u
-
```

Comment:
Note this previous ugly pattern that was an attempt to workaround a different unbound var.